api.py

```diff
@@ -0,0 +1,3 @@
+from mailbbgun import app
+
+app.run(debug=True)
```

Comment:
My preference is to never run Debug in production.  For starters, debug logging is going to consume a ton of resources.  Also, a lot of frameworks have special page rendering for errors that are great for development, but can leak a ton of information in production.  The only time I would make an exception is if we're chasing down a bug that only happens in prod that absolutely cannot be recreated in any other non-prod environments.